shared/src/models/simulated-region.ts

```diff
@@ -22,14 +22,20 @@ export class SimulatedRegion {
     public readonly type = 'simulatedRegion';
 
     /**
-     * top-left position
+     * Position of the viewport.
+     * For positive {@link size}s, this refers to the upper left corner of the simulated region, but during resizing, corners may be swapped.
      *
      * @deprecated Do not access directly, use helper methods from models/utils/position/position-helpers(-mutable) instead.
      */
     @ValidateNested()
     @IsPosition()
     public readonly position: Position;
 
+    /**
+     * Size of the viewport.
+     * The width of a viewport grows to the left, the height grows downwards.
```

Comment:
```suggestion
     * The width of a viewport grows to the right, the height grows downwards.
```

Or am I wrong again?

Reply:
No, should be correct.